### ISSUE TITLE
NullPointerException thrown on table validation

### ISSUE THREAD

**Issue by jordanpadams** [NASA-PDS/validate#591]
### Checked for duplicates

Yes - I've already checked

### 🐛 Describe the bug

When running regression tests from #519, nullpointerexception thrown

### 🕵️ Expected behavior

Error is caught and handled more gracefully.

### 📜 To Reproduce

```bash
$ validate -t src/test/resources/github469/201401031400_rdr_min_FAIL.xml

java.lang.NullPointerException: Cannot invoke "String.length()" because "line" is null
	at gov.nasa.pds.tools.validate.rule.pds4.TableValidator.validateTableDelimitedContent(TableValidator.java:243)
	at gov.nasa.pds.tools.validate.rule.pds4.TableValidator.validateDataObjectContents(TableValidator.java:160)
	at gov.nasa.pds.tools.validate.rule.pds4.TableValidator.validate(TableValidator.java:98)
	at gov.nasa.pds.tools.validate.rule.pds4.DataDefinitionAndContentValidationRule.validate(DataDefinitionAndContentValidationRule.java:86)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at gov.nasa.pds.tools.validate.rule.AbstractValidationRule.execute(AbstractValidationRule.java:64)
	at org.apache.commons.chain.impl.ChainBase.execute(ChainBase.java:191)
	at gov.nasa.pds.tools.validate.task.ValidationTask.execute(ValidationTask.java:130)
	at gov.nasa.pds.tools.validate.task.BlockingTaskManager.submit(BlockingTaskManager.java:26)
	at gov.nasa.pds.tools.label.LocationValidator.validate(LocationValidator.java:284)
	at gov.nasa.pds.validate.ValidateLauncher.doValidation(ValidateLauncher.java:1402)
	at gov.nasa.pds.validate.ValidateLauncher.processMain(ValidateLauncher.java:1688)
	at gov.nasa.pds.validate.ValidateLauncher.main(ValidateLauncher.java:1737)

PDS Validate Tool Report

Configuration:
   Version                       3.2.0-SNAPSHOT
   Date                          2023-01-31T19:20:58Z

Parameters:
   Targets                       [file:/Users/jpadams/proj/pds/pdsen/workspace/validate/src/test/resources/github469/201401031400_rdr_min_FAIL.xml]
   Severity Level                WARNING
   Recurse Directories           true
   File Filters Used             [*.xml, *.XML]
   Data Content Validation       on
   Product Level Validation      on
   Max Errors                    100000
   Registered Contexts File      /Users/jpadams/proj/pds/pdsen/workspace/validate/validate-3.2.0-SNAPSHOT/resources/registered_context_products.json



Product Level Validation Results

  FAIL: file:/Users/jpadams/proj/pds/pdsen/workspace/validate/src/test/resources/github469/201401031400_rdr_min_FAIL.xml
    Begin Content Validation: file:/Users/jpadams/proj/pds/pdsen/workspace/validate/src/test/resources/github469/201401031400_rdr_min_FAIL.tab
      ERROR  [error.table.field_value_out_of_min_max_range]   data object 2, record 9, field 32: Field has a value '11.0' that is less than the defined minimum value '12.0'.
      ERROR  [error.table.field_value_out_of_min_max_range]   data object 2, record 10, field 32: Field has a value '11.0' that is less than the defined minimum value '12.0'.
      ERROR  [error.table.field_value_out_of_min_max_range]   data object 2, record 10, field 33: Field has a value '-1' that is less than the defined minimum value '0.0'.
    End Content Validation: file:/Users/jpadams/proj/pds/pdsen/workspace/validate/src/test/resources/github469/201401031400_rdr_min_FAIL.tab
        1 product validation(s) completed

Summary:

  3 error(s)
  0 warning(s)

  Product Validation Summary:
    0          product(s) passed
    1          product(s) failed
    0          product(s) skipped

  Referential Integrity Check Summary:
    0          check(s) passed
    0          check(s) failed
    0          check(s) skipped

  Message Types:
    3            error.table.field_value_out_of_min_max_range

End of Report
Completed execution in 5799 ms
```

### 🖥 Environment Info

```bash
Mac OSX
```

### 📚 Version of Software Used

3.2.0-SNAPSHOT (issue_519 branch)

### 🩺 Test Data / Additional context

`src/test/resources/github469/201401031400_rdr_min_FAIL.xml`

### 🦄 Related requirements

N/A

### ⚙️ Engineering Details

N/A
